tests/unit/backend/layers/api/test_curation_api.py

```diff
@@ -611,7 +625,7 @@ def test__get_collection_with_dataset_failing_validation(self):
             ],
             validation_message="test message",
         )
-        res = self.app.get(f"/curation/v1/collections/{collection_version.collection_id}")
+        res = self.app.get(f"/curation/v1/collections/{collection_version.version_id}")
```

Comment:
I'm concerned about changing this without also checking that collection_id also work. Same for the other changes from collection_id to version_id

Reply:
Is this change required to make the tests pass?

Reply:
It should be working with both now. I'll double check.

Reply:
Confirmed, it works with both. I am rolling back this change.